src/ring_theory/jacobson.lean

```diff
@@ -116,4 +119,164 @@ lemma is_jacobson_iso (e : R ≃+* S) : is_jacobson R ↔ is_jacobson S :=
 
 end is_jacobson
 
+
+section localization
+open localization_map
+variables {y : R} (f : localization_map (submonoid.closure {y} : submonoid R) S)
+
+lemma not_mem_of_disjoint_closure_singleton {S : set R} :
+  disjoint ((submonoid.closure {y} : submonoid R) : set R) S → y ∉ S :=
```

Comment:
This is very specialized and probably not worth making a lemma.

A shorter proof is 
```lean
example {M : Type*} [monoid M] {S : set M} {y : M}
  (h : disjoint (↑(submonoid.closure ({y} : set M)) : set M) S) : y ∉ S :=
set.disjoint_left.1 h (subset_closure (by simp))
```